packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/container.tsx

```diff
@@ -43,9 +44,24 @@ const ConfirmResetPasswordContainer = (_: RouteComponentProps) => {
     uid: string,
     estimatedSyncDeviceCount?: number,
     recoveryKeyExists?: boolean,
-    recoveryKeyHint?: string
+    recoveryKeyHint?: string,
+    totpExists?: boolean
   ) => {
-    if (recoveryKeyExists === true) {
+    if (totpExists && recoveryKeyExists === false) {
+      navigate('/confirm_totp_reset_password', {
+        state: {
+          code,
+          email,
+          emailToHashWith,
+          estimatedSyncDeviceCount,
+          recoveryKeyExists,
+          recoveryKeyHint,
+          token,
+          uid,
+        },
+        replace: true,
```

Comment:
I think this `replace: true` is fine now that I've tested it since we just send users back to re-enter their email.